packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -167,20 +178,21 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     {
         __Ownable_init();
         transferOwnership(_owner);
+        superchainConfig = _superchainConfig;
 
         // These are set in ascending order of their UpdateTypes.
         _setBatcherHash(_batcherHash);
         _setGasConfig({ _overhead: _overhead, _scalar: _scalar });
         _setGasLimit(_gasLimit);
         _setUnsafeBlockSigner(_unsafeBlockSigner);
 
```

Comment:
The above setters emit events, the contracts being stored do not need to emit events so it is safe to use `Slot` directly altho i thought it was renamed to `Storage` in another PR?

Reply:
I put that PR on `develop` to minimize drift between it and this branch. Now I'm not sure it's working out that way.

Reply:
I implemented this using the Slot lib.